src/Tribe/Timezones.php

```diff
@@ -151,7 +159,7 @@ protected static function get_event_timestamp( $event_id, $type = 'Start', $time
 		if ( $use_event_tz || ( $use_site_tz && $site_zone_is_event_zone ) ) {
 			$datetime = get_post_meta( $event->ID, "_Event{$type}Date", true );
 
-			return strtotime( $datetime );
+			return $timestamps[ $cache_key ] = strtotime( $datetime );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 2f707d0188974c9c1690d90f2f27e7f0</sub>

Reply:
Ignoring.